update-destination.php

```diff
@@ -10,6 +10,7 @@
     <title>Update Destnation</title>
     <link rel="stylesheet" href="bootstrap.css">
     <link rel="stylesheet" href="styles.css">
+    <script src="https://cdn.ckeditor.com/4.16.2/standard/ckeditor.js"></script>
```

Comment:
Don't link the script files in the header section. While it may technically work, the standard approach is to place the script link just before the body closing tag.

Reply:
got it.